root/edit/info.tt

```diff
@@ -1,7 +1,7 @@
 [% IF summary %]
     [% IF edit.status != 1 && edit.status != 2 %]
         <div class="edit-status">
-            [%- edit.status_name -%]
+            [%- l(edit.status_name) -%]
```

Comment:
We usually use `l_...` function variants that provide translated object strings; e.g., there is `l_edit_name` (next to untranslated `edit_name`), `l_name` (next to untranslated `name`) in various classes, etc. So, for consistency, it might be better to create an `l_status_name` function and call it here.